### PR TITLE
Task3831 button group refactoring fix

### PR DESCRIPTION
**Pull request by MayaElf**
Sorry, I created new PR due to problems in the previous one after rebase.

### CODE REVIEW

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/VuetifyButtonGroupsTests.java [hunk 1]
```diff
@@ -39,13 +36,12 @@ public void mandatoryButtonGroupTest() {
...
         mandatoryButtonGroup.getButtonByIndex(1).has().css("width", "50px");
-        assertSelected(mandatoryButtonGroup.getButtonByIndex(1));
-        mandatoryButtonGroup.getButtonByIndex(2).click();
-        assertSelected(mandatoryButtonGroup.getButtonByIndex(2));
 
-        mandatoryButtonGroup.getAllButtons().forEach(HasClick::click);
-
-        assertSelected(mandatoryButtonGroup.getButtonByIndex(4));
+        mandatoryButtonGroup.is().selected(1);
+        mandatoryButtonGroup.getButtonByIndex(2).click();
```
Comment (pnatashap) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/VuetifyButtonGroupsTests.java:41]: There is no checks for getButtonByText method, should be added to improve readability

**Review by pnatashap — CHANGES_REQUESTED** [review]

**Comment by pnatashap** [comment]
@MayaElf  is this PR actual?

**Comment by MayaElf** [comment]
@pnatashap Natalia, yes, this actual PR.

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/VuetifyButtonGroupsTests.java (hunk 1 above)
Comment (MayaElf) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/VuetifyButtonGroupsTests.java:41]: For these checks another task was created https://github.com/jdi-testing/jdi-light/issues/4107

**Pull request merged**